lewis/devices/julabo/device.py

```diff
@@ -20,10 +20,29 @@
 from __future__ import print_function
 from collections import OrderedDict
 from lewis.devices import StateMachineDevice
+from lewis.core.utils import check_limits
 from . import states
 
 
 class SimulatedJulabo(StateMachineDevice):
+    internal_p = 0.1  # The proportional
+    internal_i = 3  # The integral
+    internal_d = 0  # The derivative
+    external_p = 0.1  # The proportional
+    external_i = 3  # The integral
+    external_d = 0  # The derivative
+    temperature_low_limit = 0.0  # Usually set in the hardware
+    temperature_high_limit = 100.0  # Usually set in the hardware
+    set_point_temperature = 24.0  # Set point starts equal to the current temperature
+    heating_power = 5.0  # The heating power
+    version = "JULABO FP50_MH Simulator, ISIS"
+    status = "Hello from the simulated Julabo"
+    is_circulating = 0  # 0 for off, 1 for on
+    temperature = 24.0  # Current temperature in C
+    external_temperature = 26.0  # External temperature in C
+    circulate_commanded = False
+    temperature_ramp_rate = 5.0  # Guessed value in C/min
+
     def _initialize_data(self):
```

Comment:
This is completely empty now and can be removed. As lewis is evolving it might not even be necessary at all in the (very near) future, so the way you defined the members as part of the class is fine. Everything that lewis allows you to do now (namely overriding those values in a setup) is compatible with the way you wrote it here.